22-discovery.ks

```diff
@@ -15,8 +15,8 @@ cat > /etc/NetworkManager/NetworkManager.conf <<'NM'
 [main]
 monitor-connection-files=yes
 no-auto-default=*
-#[logging]
-#level=DEBUG
+[logging]
+level=DEBUG
```

Comment:
Won't hurt, the level of detail is decent yet not spamming with trace messages. Journald is configured to use 20MB of memory for volatile backlog.